src/caps.rs

```diff
@@ -1,12 +1,12 @@
 //! A set of standard names for capabilities that can be provided by a host
-pub const MESSAGING: &str = "wasmcloud:messaging";
-pub const KEY_VALUE: &str = "wasmcloud:keyvalue";
-pub const HTTP_SERVER: &str = "wasmcloud:httpserver";
-pub const HTTP_CLIENT: &str = "wasmcloud:httpclient";
 pub const BLOB: &str = "wasmcloud:blobstore";
+pub const HTTP_CLIENT: &str = "wasmcloud:httpclient";
+pub const HTTP_SERVER: &str = "wasmcloud:httpserver";
+pub const KEY_VALUE: &str = "wasmcloud:keyvalue";
+pub const MESSAGING: &str = "wasmcloud:messaging";
 pub const EVENTSTREAMS: &str = "wasmcloud:eventstreams";
-pub const EXTRAS: &str = "wasmcloud:extras";
-pub const LOGGING: &str = "wasmcloud:logging";
+pub const EXTRAS: &str = "wasmcloud:builtin:extras";
```

Comment:
this is actually `wasmcloud:builtin:numbergen`

Reply:
Should probably also change the display on this contract from "extras" to "Number Generation"

Reply:
see https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/web_assembly/imports.ex#L5

Reply:
good catch, thanks